### PR TITLE
Update cURL client guide

### PR DESCRIPTION
**Pull request by noahcrowley**

### CODE REVIEW

File: _pages/clients/curl.md [hunk 1]
```diff
@@ -15,78 +15,171 @@ redirect_from:
...
 {% endhighlight %}
 
-#### Upload File
+Response: `{"error":{"message":"algorithm algo://util/whoopsWrongAlgo/ not found"}}`
+
+You can read more about [Error Handling](/developers/algorithm-development/algorithm-errors) in the [Algorithm Development](/developers/algorithm-development) section of the dev center.
+
+### Limits
+
+Your account can make up to {{site.data.stats.platform.max_num_algo_requests}} Algorithmia requests at the same time (this limit <a onclick="Intercom('show')">can be raised</a> if needed).
```
Comment (bnlb) [inline _pages/clients/curl.md]: `Intercom` is undefined in the global scope in prod. Let's change the link slightly:
```suggestion
Your account can make up to {{site.data.stats.platform.max_num_algo_requests}} Algorithmia requests at the same time (this limit <a href="mailto:support@algorithmia.com?subject=Request to raise max algorithm request limit">can be raised</a> if needed).
```

**Review by bnlb — CHANGES_REQUESTED** [review]
Only blocker is that Intercom link which is a quick update. Everything else looks good 👍

**Review by stevielkim — DISMISSED** [review]

**Comment by stevielkim** [comment]
@noahcrowley can you see what changes need to be made and finish this one up please? Thanks!

**Comment by noahcrowley** [comment]
Should be good to go, thanks!

**Comment by stevielkim** [comment]
@bnlb can you review this PR please? it's not a big one, but requires your review thanks!

**Review by bnlb — APPROVED** [review]
Thanks for the reminder @StephanieKim. lgtm!

**Pull request merged**